packages/components/src/List/ListComposition/collection.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
`import/no-extraneous-dependencies` is disabled for `.stories.js` and `.test.js` :/
This file is a data file for stories